cdp_scrapers/instances/portland.py

```diff
@@ -134,8 +144,65 @@ def get_matter(self, event_page: BeautifulSoup) -> Optional[Matter]:
         #   The first description phrase in each item on agenda page.
         #   This is usually hyperlinked to another page with more details.
 
+        # Find document number
+        doc_number_element_sibling = event_page.find(
+            "div", text=re.compile("Document number"), attrs={"class": "field__label"}
+        )
+
+        # If there is no document number, skip this minute item
+        if doc_number_element_sibling is None:
+            return None
+        doc_number_element = doc_number_element_sibling.next_sibling
+        doc_number = doc_number_element.find("div", class_="field__item").text.strip()
+
+        # Find title
+        title_div = event_page.find("div", class_="council-document__title")
+        matter_title = title_div.find("a").text.strip()
+
+        # Find type
+        title_div.find("a").clear()
+        matter_type = title_div.text.strip()
+        if matter_type[0] == "(" and matter_type[-1] == ")":
+            matter_type = matter_type[1:-1]
+
+        # Find result status
+        result_status_element_sibling = event_page.find(
+            "div", text=re.compile("Disposition"), attrs={"class": "field__label"}
+        )
+        result_status_element = result_status_element_sibling.next_sibling
+        result_status = result_status_element.text
+        if result_status in ["Accepted", "Passed"]:
+            result_status = MatterStatusDecision.ADOPTED
+        elif "Passed to" in result_status:
+            result_status = MatterStatusDecision.IN_PROGRESS
+        else:
+            result_status = None
```

Comment:
I guess we still haven't seen an example string we can use to decide `MatterStatusDecision.REJECTED`. Is that correct Shak?

Reply:
No, I didn't find any example of a proposed piece of legislation being rejected. I did find one case in which 1 commissioner voted "Nay", but the legislation still passed.